js/debug/job_recommendation.js

```diff
@@ -9,7 +9,7 @@
             if(clicks == null)
                 clicks = {};
 
-            clicks[jobId] = {'recommender': recommender, 'timestamp': new Date().getTime()};
+            clicks[jobId] = {recommender: recommender, timestamp: new Date().getTime(), page: window.location.pathname};
```

Comment:
Will this be hard to report?  We'll have lots of pages that look like this in the future:

/jobs/foo-123
/jobs/test-123

Will it be easy to filter by paths that start with `/jobs/`?

Reply:
@johnnyoshika If we send the full path `/jobs/foo-123` it would be easy to filter paths that start with `/jobs/` in Application Insights.  

The other option would be to check to see if the path starts with `/jobs/` in JavaScript and only send that part of the path.  

Do you have any preference as I am fine with both.